README.md

```diff
@@ -54,6 +54,7 @@ require('cmake').setup({
   cmake_executable = 'cmake', -- CMake executable to run.
   save_before_build = true, -- Save all buffers before building.
   parameters_file = 'neovim.json', -- JSON file to store information about selected target, run arguments and build type.
+  default_parameters = { run_dir = '', args = {}, build_type = '' }, -- The default values in `parameters_file`.
```

Comment:
Default build type was `Debug`. Could you keep it here and in `lua/cmake/config.lua`?